### PR TITLE
Zustand

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
Add Zustand Global state container.
Connect Snack Bar message to zustand.
Add Snack Message zustand test.

https://olegmoshkovich.github.io/Share/share/v/p/index.ifc#c:-144.36,14.11,147.82,-40.42,17.84,-2.28

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +46,7 @@
     "@testing-library/dom": "^8.11.1",
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^12.1.2",
+    "@testing-library/react-hooks": "^12.1.2",
```
Comment (pablo-mayrgundter) [inline package.json:49]: Per discussion, this ought to be ^8.0.0 and with that, tests pass for me.  w00t!

Comment (OlegMoshkovich) [reply package.json:49]: yep

File: src/Containers/CadView.jsx [hunk 2]
```diff
@@ -65,8 +66,9 @@ export default function CadView({
   const [isItemPanelOpen, setIsItemPanelOpen] = useState(false)
   const isItemPanelOpenState = {value: isItemPanelOpen, set: setIsItemPanelOpen}
   const [isLoading, setIsLoading] = useState(false)
-  const [loadingMessage, setLoadingMessage] = useState()
   const [model, setModel] = useState(null)
+  const snackMessage = useStore((state)=>state.snackMessage)
```
Comment (pablo-mayrgundter) [inline src/Containers/CadView.jsx:70]: spaces around arrow.  Isn't this flagged by eslint?

**Review by pablo-mayrgundter — APPROVED** [review]
LGTM.  Maybe consider splitting the big state object into slices. https://github.com/pmndrs/zustand/wiki/Splitting-the-store-into-separate-slices

**Pull request merged**